public/js/thirdpartyservices.js

```diff
@@ -92,5 +92,15 @@ var thirdPartyServicesJsonArray = [
       privacy: 'https://clicky.com/terms#privacy',
       terms: ''
     }
+  },
+  {
+    name: 'Flurry Analytics',
+    model: 'flurry',
+    flurry: false,
+    logo: 'images/flurry.png',
+    link: {
+      privacy: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/',
```

Comment:
From what I can check online, flurry's privacy policy link is this

https://privacy.oath.com

From the [terms and conditions link](https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html)
> guidelines incorporated in the Terms of Service by reference, including, but not limited to, Flurry’s Privacy Policy located at https://privacy.oath.com (“Privacy Policy”) (or such other URL that Flurry may provide from time to time) (collectively, the "Agreement").

Reply:
Hi @nisrulz 

You're right. I've used the url that one of my clients is using in his app.

Thanks!